lib/grp.gi

```diff
@@ -433,10 +439,12 @@ InstallMethod( IsNilpotentGroup,
 ##
 #M  IsPerfectGroup( <G> ) . . . . . . . . . . . .  test if a group is perfect
 ##
-InstallImmediateMethod( IsPerfectGroup,
-    IsSolvableGroup and HasIsTrivial,
-    0,
-    IsTrivial );
+#  This used to be an immediate method. It was replaced by a TrueMethod (in
+#  the gd file)
+#InstallImmediateMethod( IsPerfectGroup,
+#    IsSolvableGroup and HasIsTrivial,
+#    0,
+#    IsTrivial );
```

Comment:
Let's just remove this completely -- see also my suggestion above for the other "missing" direction.

Reply:
ditto OK